index.d.ts

```diff
@@ -46,20 +46,20 @@ import { FastImageProps } from "react-native-fast-image";
 import { theme as themeDef } from "./src/theme";
 import { BUTTON_VARIANTS } from "./src/components/Button";
 
-type addPrefixToObject<
-  TObject extends object,
+type addPrefixToo<
+  To extends o,
   TPrefix extends string
-> = `${TPrefix}${keyof TObject}`;
+> = `${TPrefix}${keyof To}`;
 
 const theme: {
   colors: {
-    font: addPrefixToObject<typeof themeDef.colors.font, "font.">;
-    background: addPrefixToObject<
+    font: addPrefixToo<typeof themeDef.colors.font, "font.">;
+    background: addPrefixToo<
       typeof themeDef.colors.background,
       "background."
     >;
-    border: addPrefixToObject<typeof themeDef.colors.border, "border.">;
-    toast: addPrefixToObject<typeof themeDef.colors.toast, "toast.">;
+    border: addPrefixToo<typeof themeDef.colors.border, "border.">;
```

Comment:
@kiran-kumar011 Looks like `Object` got replaced by `o` in many places, Please check.